components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/impl/DASLoginDataPublisherImpl.java

```diff
@@ -265,4 +269,16 @@ private String getCommaSeparatedUserRoles(String userName, String tenantDomain)
         return StringUtils.EMPTY;
     }
 
+    /**
+     * Method to check if local idp was used for authentication.
+     * @param authenticationData
+     * @return boolean value to indicate if authenticated idp is local idp.
+     */
+    private boolean hasLocalIdpUsedForAuthentication(AuthenticationData authenticationData) {
```

Comment:
This method (at least the method description) doesn't convey the message that this is an authentication which took place using an existing local session. (no actual authentication took place. Instead it's just allowing access using an existing session)

Reply:
Fixed with https://github.com/wso2-extensions/identity-data-publisher-authentication/pull/73/commits/df10e1bad72a2c3392ca71df2f863d6603df1d93

Please note that this method contains check for both first time and existing sessions. Therefore I have mentioned in comment as you explained.